### PR TITLE
Bump version to v1.0.0

### PR DESCRIPTION
**Pull request by jepemi**

### CODE REVIEW

**Review by irenebm — CHANGES_REQUESTED** [review]

File: versions.md [hunk 1]
```diff
@@ -19,6 +15,16 @@ This release includes the following **features**:
 * `StatisticsBackend::get_status_data` returns an specific status data of a given id.
 * Get physical information from discovery packets.
 
+This release includes the following **dependencie update**:
+|  | Repository | Old Version | New Version |
+|---|---|---|---|
+| Foonathan Memory Vendor | [eProsima/foonathan_memory_vendor](https://github.com/eProsima/foonathan_memory_vendor) | [v1.3.1](https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1) | [v1.3.1](https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1) |
```
[the remark this answers is not in the document]
Comment (jepemi) [reply versions.md:21]: Right, to be removed.

File: docs/rst/types/status_kind.rst [hunk 2]
```diff
@@ -64,31 +63,37 @@ describes which |StatusKind-api| each of these :ref:`entities<types_entity_kind>
...
 | |INCOMPATIBLE_QOS-api|        | |OK-api|/|ERROR-api|  |
 +-------------------------------+-----------------------+
-| |INCONSISTENT_TOPIC-api|      | \-                    |
-+-------------------------------+-----------------------+
 | |LIVELINESS_LOST-api|         | |OK-api|/|WARNING-api||
 +-------------------------------+-----------------------+
 | |LIVELINESS_CHANGED-api|      | |OK-api|              |
 +-------------------------------+-----------------------+
-| |DEADLINE_MISSED-api|         | |OK-api|/|WARNING-api||
+| |DEADLINE_MISSED-api|         | |OK-api|/|ERROR-api|  |
```
[the remark this answers is not in the document]
Comment (jepemi) [reply docs/rst/types/status_kind.rst:88]: Undo changes

File: docs/rst/types/status_kind.rst [hunk 3]
```diff
@@ -64,31 +63,37 @@ describes which |StatusKind-api| each of these :ref:`entities<types_entity_kind>
...
 +-------------------------------+-----------------------+
-| |SAMPLE_LOST-api|             | |OK-api|/|WARNING-api||
+| |SAMPLE_LOST-api|             | |OK-api|/|ERROR-api|  |
```
[the remark this answers is not in the document]
Comment (jepemi) [reply docs/rst/types/status_kind.rst:90]: Undo changes

**Review by irenebm — APPROVED** [review]
LGTM

File: docs/rst/notes/notes.rst [hunk 4]
```diff
@@ -30,28 +34,31 @@ This release includes the following **dependencies update**:
...
     *   - Foonathan Memory Vendor
         - `eProsima/foonathan_memory_vendor <https://github.com/eProsima/foonathan_memory_vendor>`_
-        - `v1.3.0 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.0>`_
+        - `v1.3.1 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1>`_
         - `v1.3.1 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1>`_
     *   - Fast CDR
         - `eProsima/Fast-CDR <https://github.com/eProsima/Fast-CDR>`_
-        - `v1.0.27 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.0.27>`_
         - `v1.1.0 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.1.0>`_
+        - `v2.1.1 <https://github.com/eProsima/Fast-CDR/releases/tag/v2.1.1>`_
```
Comment (rsanchez15) [inline docs/rst/notes/notes.rst:42]: I think we should target [Fast CDR v2.1.2](https://github.com/eProsima/Fast-CDR/releases/tag/v2.1.2). Update this everywhere

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

**Review by rsanchez15 — APPROVED** [review]

**Pull request merged**